test/fluree/db/query/json_ld_compound_test.clj

```diff
@@ -118,7 +118,7 @@
                                  :select  [?name ?favNums]
                                  :where   {:schema/name ?name
                                            :ex/favNums  ?favNums}
-                                 :orderBy (desc ?favNums)})
+                                 :orderBy [(desc ?favNums)]})
```

Comment:
Does this change require ordering clauses to be wrapped in a vector now?

Reply:
Did you change it? If so, could you change this test back too?

Reply:
This still needs to be done, it looks like?